x-pack/filebeat/module/crowdstrike/_meta/config.yml

```diff
@@ -0,0 +1,8 @@
+- module: crowdstrike
+  # All logs
+  falcon:
+    enabled: true
```

Comment:
I did some testing. This looks like it needs to be true. You can see this in other modules, like [cisco](https://github.com/elastic/beats/blob/master/x-pack/filebeat/module/cisco/_meta/config.yml#L3) or [googlecloud](https://github.com/elastic/beats/blob/master/x-pack/filebeat/module/googlecloud/_meta/config.yml#L3). I did a quick test and set these to false, then ran ```./filebeat modules enable crowdstrike```. It didn't collect the data until I changed ```modules.d/crowdstrike.yml``` to set enabled to true. Leaving it as is so that when a user runs ```./filebeat modules enable crowdstrike``` this works without having to edit enabled to set it to true.